sphinx/writers/latex.py

```diff
@@ -2308,6 +2310,7 @@ def visit_literal_block(self, node):
                 hlcode += '\\end{sphinxVerbatimintable}'
             else:
                 hlcode += '\\end{sphinxVerbatim}'
+            self.body.append('\n' + hllines)
             self.body.append('\n' + hlcode + '\n')
```

Comment:
@mgeier I recall you had once some problem with the way we add Verbatim to body. Would

    self.body.append('\n' + hllines + '\n' + hlcode + '\n')

be ok for your extension nbsphinx ?

Reply:
@jfbu Thanks for remembering this issue!

Yes, this should be fine, because the `SphinxVerbatim` environment is still contained in a single list element which is the last one appended to `self.body`.
It doesn't matter if `hllines` is a separate list item or if `hllines` and `hlcode` are concatenated into one list item.
For reference, this was the previous issue you were mentioning: https://github.com/spatialaudio/nbsphinx/issues/78 and #3224.

After that issue, I've started using `pushbody()` and `popbody()`, which makes the whole thing much more robust.
Since then, it doesn't matter at all how the partial strings are appended to `self.body`.

Reply:
@mgeier thanks for explanations, I will push a commit using single list element formed from `hllines` and `hlcode`.